Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -437,9 +437,21 @@ static ErrorOr<void> build_macroblocks(JPEGLoadingContext& context, Vector<Macro
             for (u8 hfactor_i = 0; hfactor_i < scan_component.component.hsample_factor; hfactor_i++) {
                 // A.2.3 - Interleaved order
                 u32 mb_index = (vcursor + vfactor_i) * context.mblock_meta.hpadded_count + (hfactor_i + hcursor);
```

Comment:
only vaguely related: could we call this `macroblock_index` instead of `mb_index`?

Reply:
We should even get rid of this variable in a big refactoring, but let's do that until then.